Update docs with SSO users IdP claims (#2843)
commercetools/merchant-center-application-kit: website/src/content/api-reference/commercetools-frontend-application-shell-connectors.mdx

}
```

> For users who have logged-in into the Merchant Center using SSO, there will be another object property (**idTokenUserInfo**) which will contain some standard claims from the id token we get in our backend when processing the OpenId Connect (OIDC) flow with their Identify Provider.

Reviewer: ```suggestion
If the user logged into the Merchant Center using [Single Sign-On](https://docs.commercetools.com/merchant-center/single-sign-on-beta), we expose an additional property `idTokenUserInfo` which contains some of the user info claims (OpenID Connect) from the user's Identity Provider.
```

Author: Thanks for the suggestion.

Updated in this commit: 6898054f37f9176a6f1fadd0c0766f4d64454c6b

> ```json

Reviewer: ````suggestion
```json highlightedLines="14-22"
````

Author: Updated in this commit: 6898054f37f9176a6f1fadd0c0766f4d64454c6b

{
  "user": {
    "id": "3mj76c04-f910-4223-84e1-f97b0fe291c2",
    "email": "aHR0cHM6Ly9kZXYt0udXMuYXVLzphdXRoiMDljMjYzZWQwOTg4MmU2OGU=@3241a8e3-cc17-4e7e-b6vz-1d0fdb.sso",
    "firstName": "John",
    "lastName": "Doe",
    "businessRole": "Consultant",
    "locale": "en-GB",
    "timeZone": "Europe/Berlin",
    "projects": [
      { "key": "my-project-A", "name": "My Project A" },
      { "key": "my-project-B", "name": "My Project B" }
    ],
    "idTokenUserInfo": {
      "iss": "https://test-oidc.auth0.com/",
      "sub": "1d9ca1aa-459c-48bf-bbf2-ca7bb0a6deaa",

>       "aud": "https://mc-api.europe-west1.gcp.escemo.com",

Author: Not completely sure if this is a good example value.
Any suggestion would be much appreciated.

>       "iss": "https://test-oidc.auth0.com/",
>       "sub": "1d9ca1aa-459c-48bf-bbf2-ca7bb0a6deaa",
>       "aud": "https://mc-api.europe-west1.gcp.escemo.com",

Reviewer: I would just put random values here. Could also be some placeholder value

```suggestion
      "iss": "<the-issuer>,
      "sub": "<the-subject>",
      "aud": "<the-audience>",
```

Author: Updated in this commit: 6898054f37f9176a6f1fadd0c0766f4d64454c6b
